### ISSUE TITLE
General server performance degradation from 2.2.8 to 2.2.10 update

### ISSUE THREAD

**Issue by Tommeh1** [GTNewHorizons/GT-New-Horizons-Modpack#12639]
### Your Pack Version

2.2.10

### Your Server

Private Server

### Java Version

Java 19

### Type of Server

Vanilla Forge

### Your Expectation

/

### The Reality

With 2 to 3 players moving around in base and ME constantly crafting things, performance would hold reasonably stable around 16 to 17TPS, since the update with nothing special going on this has suddenly declined to 11 to 12TPS, snapshots are attached below.
[Snapshots.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/10775027/Snapshots.zip)

### Your Proposal

/

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Eranicus** [comment]
I don't play on server, but I also experience overall worse tps performance in single.  

A while ago i was testing 2.2.9-3dev vs 2.2.9-4dev and the problem started in gregtech-1.7.10-5.09.41.219-pre , im not sure if it was fixed/optimised or reverted since then.

https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12336

**Comment by Tommeh1** [comment]
Your performance decrease is neglectable, but i'm talking about maybe 20ms or so if not more extra per tick

**Comment by Eranicus** [comment]
> Your performance decrease is neglectable, it is within a margin of error, but i'm talking about maybe 20ms or so if not more extra per tick

I tested my base in idle mode. 2,2,8 average is 9/50 ms after update it was average 10.2-10.5/50. I would not call +10% margin of error. 

Anyway I just pointed to a "possible" problem... it might be something else.

**Comment by Tommeh1** [comment]
And for comparison sake to my current snapshot the snapshot from issue #12520 can be used.
Not too much has changed since than, either in terms of machines etc or overall performance until right before the update.

**Comment by chochem** [comment]
I measured over a minute in 2.2.8 and 2.2.10 and my average tick time was 2% worse in 2.2.10. So practically the same. you might have to look a bit more at your specific save and see what is causing in.

**Comment by Tommeh1** [comment]
For one whatever an advanced EBF is exactly is spending about all its time on AE2 stuff: 
![image](https://user-images.githubusercontent.com/39203645/219915099-13984427-7fa3-4ea6-b35d-4440a6f07652.png)
Same story here it seems
![image](https://user-images.githubusercontent.com/39203645/219915328-d2d87520-6d38-4ec2-a24b-23e6d38fd66c.png)
Or here:
![image](https://user-images.githubusercontent.com/39203645/219915697-6e45158a-e9af-45e3-901f-3e669be6a7b5.png)
In short whatever that is is taking up a massive chunk of time on top of `appeng.me.Grid.update ()` constantly eating about 16%, hence my previous comment somewhere on poor AE2 related performance.

**Comment by chochem** [comment]
advanced EBF would be Volcanus. I have those too though, mine work fine. Maybe its stocking input busses?

**Comment by Tommeh1** [comment]
> advanced EBF would be Volcanus. I have those too though, mine work fine. Maybe its stocking input busses?

If that performance hit is from all our stocking input busses than i would highly recommend optimizing them if you see how much of our performance they're taking up.

**Comment by Tommeh1** [comment]
It turns out that this big hit from the update was coming from the java9+ stuff, putting the server back on java8 heavily increased performance again.

**Comment by Tommeh1** [comment]
Fixed in 2.3.2
